src/test/fuzz/script.cpp

```diff
@@ -15,20 +15,36 @@
 #include <script/standard.h>
 #include <streams.h>
 #include <test/fuzz/fuzz.h>
+#include <univalue.h>
 #include <util/memory.h>
 
 void initialize()
 {
     // Fuzzers using pubkey must hold an ECCVerifyHandle.
     static const auto verify_handle = MakeUnique<ECCVerifyHandle>();
+
+    SelectParams(CBaseChainParams::REGTEST);
 }
 
 void test_one_input(const std::vector<uint8_t>& buffer)
 {
     const CScript script(buffer.begin(), buffer.end());
 
     std::vector<unsigned char> compressed;
-    (void)CompressScript(script, compressed);
+    if (CompressScript(script, compressed)) {
+        const unsigned int size = compressed[0];
+        assert(size >= 0 && size <= 5);
+        CScript decompressed_script;
+        const bool ok = DecompressScript(decompressed_script, size, compressed);
+        assert(ok);
+    }
+
+    for (unsigned int size = 0; size < 6; ++size) {
+        std::vector<unsigned char> vch(GetSpecialScriptSize(size), 0x00);
+        vch.insert(vch.end(), buffer.begin(), buffer.end());
+        CScript decompressed_script;
+        (void)DecompressScript(decompressed_script, size, vch);
+    }
```

Comment:
Looks like you accidentally added a unit test here? This does not depend on any fuzz input.

Reply:
@MarcoFalke Are you sure about that? `vch` is built from `buffer` which is built from fuzz input, no? :)

Reply:
`buffer` is only *appened* to `vch`, but pratically never read. Only the beginning of `vch` (20 or 32 bytes) are read, and they are all constant (zero). So this does never read any fuzz input, except for copying into the vector.

I suggest to remove this for loop and content.

Reply:
ping @practicalswift

Reply:
I'll fix. Thanks for the ping :)